src/components/EventMap.js

```diff
@@ -162,57 +138,41 @@ export default function(store){
 
       openPopupsOnClick() {
 
-        ['unclustered-points', 'stars'].forEach(layer => {
```

Comment:
Some of these diffs look sketchier than they are, here for example is just unwrapping the contents of the forEach, since there is now only one layer, `points`.